### DISCUSSION TITLE
guide/custom-style/

### DISCUSSION THREAD

**Discussion by giscus** [pengzhanbo/vuepress-theme-plume#205] [docs-comment]
# guide/custom-style/

主题定制 支持自定义样式。 主题虽然使用 SASS 作为 CSS 预处理器，但所有的颜色使用的是 CSS Vars 定义， 因此，你可以创建 一个 css 文件 或 scss 文件，进行覆盖。 首先，在 .vuepress 目录中，创建一个 styles/index.css 文件， 然后在 客户端配置文件 中，引入该文件即可。 Style 文件 在 ....

https://theme-plume.vuejs.press/guide/custom-style/

**Comment by bglove** [comment]
我用上述方式在config.js中引入我自己写的css文件，但并没有引入成功。我使用的是第一种方式，主要想改变li标签的行高。这是我的代码
:root li{
    line-height: 32px!important;
}
但是没有生效

**Reply by pengzhanbo** [reply]
不是在 config.js 中引入，而是在 client.js 中引入。

**Reply by bglove** [reply]
十分感谢，已经生效了。是我自己没有仔细阅读文档，不好意思！
